airtable/plugin.py

```diff
@@ -1,12 +1,42 @@
 import json
 from dataclasses import dataclass
 from datetime import datetime
-from typing import Any
+from typing import Any, Tuple
+import re
 
 import httpx
 
 from lutraai.augmented_request_client import AugmentedTransport
 
+@dataclass
+class AirtableBaseID:
+    id: str
+
+@dataclass
+class AirtableTableID:
+    id: str
```

Comment:
Does using names still work?  Perhaps this should be called `id_or_name`?  Relatedly, I just made https://github.com/d8e-ai/alda/issues/2469, which might be helpful too.

Reply:
i think we should use id here and add a doc string to the class

Reply:
Added the docstring and code_gen was able generate code used a name